docs/scenarios_authoring/TestingAndDebugging.md

```diff
@@ -4,18 +4,26 @@ sidebar_position: 8
 
 # Testing and Debugging
 
-There are several features in Nussknacker designed specifically to ease testing and debugging; this page provides a brief explanation how to use them. This functionality is grouped in the Test panel docked in  the right side of the Designer canvas; if you did not rearrange panels it is in the right part of the Designer canvas. 
+There are several features in Nussknacker designed specifically to ease testing and debugging; this page provides a brief explanation how to use them. This functionality is grouped in the Test panel docked on the right side of the Designer canvas; if you did not rearrange panels it is in the right part of the Designer canvas. 
```

Comment:
last sentence contains repeated content; can be shortened (?) to:

This functionality is grouped in the Test panel;  if you did not rearrange panels it is in the right part of the Designer canvas.